Code/Engine/Foundation/Math/Implementation/Quat_inl.h

```diff
@@ -132,6 +141,11 @@ bool ezQuatTemplate<Type>::IsNaN() const
 template <typename Type>
 bool ezQuatTemplate<Type>::IsEqualRotation(const ezQuatTemplate<Type>& qOther, float fEpsilon) const
 {
+  if (v.IsEqual(qOther.v, 0.00001f) && ezMath::IsEqual(w, qOther.w, 0.00001f))
```

Comment:
Use fEpsilon instead of hardcoded value here?

Reply:
fEpsilon is used to compare the angle between the two quaternions. The first if however just checks if the quaternions are the same. Thus a different epsilon value.